utils/data_helpers.py

```diff
@@ -282,22 +328,25 @@ def encode_tabular_data(tab_xray_df):
     for column in tab_xray_df:
         if column not in ignores and column not in normalises:
             # One-hot encode the categorical data
-            one_hot_subset_df = pd.get_dummies(tab_xray_df[column])
-            one_hot_subset_df = one_hot_subset_df.add_prefix(f'{column}_')
+            one_hot_subset_df = pd.get_dummies(tab_xray_df[column], prefix=column)
+            # one_hot_subset_df = one_hot_subset_df.add_prefix(f'{column}_')
```

Comment:
just checking, meant to be commented out?

Reply:
Yeah - I'll delete the comment. Nice spot